modules/prometheus-cpp/1.2.4/presubmit.yml

```diff
@@ -0,0 +1,21 @@
+matrix:
+  platform:
+  - debian10
+  - ubuntu2004
+  - macos
+  - macos_arm64
+  - windows
+  bazel:
+  - 7.x
+  - 6.x
+tasks:
+  verify_targets:
+    name: Verify build targets
+    platform: ${{ platform }}
+    bazel: ${{ bazel }}
+    build_flags:
+    - '--define=with_civetweb_ssl=true'
```

Comment:
I’d remove the SSL config until Bazel or the Mac Runners are fixed.

Reply:
OK